liblouis/maketable.c

```diff
@@ -96,6 +96,7 @@ printRule(TranslationTableRule *rule, widechar *rule_string) {
 		for (int k = 0; k < rule->charslen; k++) rule_string[l++] = rule->charsdots[k];
 		rule_string[l++] = ' ';
 		for (int k = 0; k < rule->dotslen; k++)
+			// assume that _lou_getCharFromDots finds a char
```

Comment:
isn't this a bit scary? You potentially insert '\0' in the `rule_string`.

Reply:
Yeah well I didn't really know what to do with it. We could check for NULL and throw an error?

Reply:
Replacing it with a space (old behavior) is also not great.

Reply:
I thought if you insert '\0' you basically truncate the `rule_string`. Throwing an error might be better.

Reply:
OK

Reply:
@egli see https://github.com/liblouis/liblouis/pull/874/commits/60f8857e894530e1eae2170530111f83e59e589f